019_t_L54_safe.js

```
@@ -0,0 +1,361 @@
+          if(socket.room === undefined) {
+            socket.room = 'multiverse'; // store the room name in the socket session for this client
+          }
+          usernames.push({name: socket.username, room: socket.room});
+          var query = new Array();
+          query.title = "r "+socket.room;
+          if(socket.room === 'multiverse') {
+            changeRoom(query);
+          }
+        } else { // if it's a EXISTING user
+          socket.room = usernames[existing].room;
+          var query = new Array();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'query'. Consider to delete it or rename it.

'query' is defined here:


(lint/suspicious/noRedeclare)